setup.py

```diff
@@ -316,6 +317,7 @@ def build_extension(self, ext):
             _config = "{0}Mono".format(CONFIG)
             _solution_file = "pythonnet.sln"
             _custom_define_constants = False
+            defines.append("NET40")
```

Comment:
This is new, is this intentional?

Reply:
@filmor yes, it is. This is used in the same commit.